### DISCUSSION TITLE
Welcome to rn-foreground-service Discussions!

### DISCUSSION THREAD

**Discussion by Raja0sama** [Raja0sama/rn-foreground-service#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by satyamkumarjha9966** [comment]
I Got This Error While Start The Service

possible unhandled promises rejection (id: 14); typeError: cannot read property "startService" of null

![WhatsApp Image 2023-12-27 at 3 53 34 PM](https://github.com/Raja0sama/rn-foreground-service/assets/123254088/79d500a3-219e-4472-8663-12c6c3d99203)

**Comment by Domesan-U** [comment]
Does it work for Android 14 ? i am facing issue when i moved it to android 14. The task is not running the function but the task is actually running. 
ReactNativeForegroundService.add_task(() => update(),   //This update() function is not running
 {
  delay: 1000,
  onLoop: true,
  taskId: "taskid",
  onError: (e) => console.log(`Error logging:`, e),
});

**Reply by CodeSyncr** [reply]
Same issue here
